### PR TITLE
Add merged label to ghstack prs

### PR DESCRIPTION
**Pull request by clee2000**
not very elegant

other option might be adding something to pytorchbot to listen to push events for master?

### CODE REVIEW

**Comment by kit1980** [comment]
Let's not change this until January so we have a stable foundation for this half's contributions metric.
Then in January we can update this and backfill one month.

**Comment by kit1980** [comment]
Also need a similar to revert, propagated from bottom of the stack to top.

**Comment by clee2000** [comment]
> Also need a similar to revert, propagated from bottom of the stack to top.

afaik you still have to revert each pr in a stack one by one so it shoudnt be affected

File: .github/scripts/trymerge.py [hunk 1]
```diff
@@ -871,8 +871,9 @@ def merge_ghstack_into(
         skip_mandatory_checks: bool,
         comment_id: Optional[int] = None,
         land_check_commit: Optional[str] = None
-    ) -> None:
+    ) -> List["GitHubPR"]:
         assert self.is_ghstack_pr()
+        additional_prs = []
```
Comment (malfet) [inline .github/scripts/trymerge.py:876]: Nit
```suggestion
        additional_prs:List["GitHubPR"] = []
```

**Review by malfet — APPROVED** [review]

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**